azure/backgroundJobTemplate.json

```diff
@@ -0,0 +1,177 @@
+      }
+    },
+    "railsEnv": {
+      "type": "string",
+      "defaultValue": "production",
+      "metadata": {
+        "description": "Environment for the rails app."
+      }
+    },
+    "databaseName": {
+      "type": "string",
+      "metadata": {
+        "description": "The name of the database that the app will connect to."
+      }
+    },
+    "databaseServerHostName": {
+      "type": "string",
+      "metadata": {
+        "description": "The fqdn of the psql server hosting the database."
+      }
+    },
+    "databasePort": {
+      "type": "string",
+      "metadata": {
+        "description": "The default port for the psql server."
+      }
+    },
+    "databaseUsername": {
+      "type": "string",
+      "metadata": {
+        "description": "The username used to connect to the database."
+      }
+    },
+    "databasePassword": {
+      "type": "string",
+      "metadata": {
+        "description": "The password used to connect to the database."
+      }
+    },
+    "secretKeyBase": {
+      "type": "string",
+      "metadata": {
+        "description": "Secret Key Base."
+      }
+    },
+    "sentryDSN": {
+      "type": "string",
+      "metadata": {
+        "description": "Connection string for Sentry monitoring."
+      }
+    },
+    "settingsAuthenticationSecret": {
+      "type": "string",
+      "metadata": {
+        "description": "Auth secret for manage back-end."
+      }
+    },
+    "settingsManageApiSecret": {
+      "type": "string",
+      "metadata": {
+        "description": "The secret shared between manage courses backend and manage courses API to authorize requests."
+      }
+    },
+    "settingsSearchApiSecret": {
+      "type": "string",
+      "metadata": {
+        "description": "The secret shared between manage courses backend and search and compare API to authorize requests."
+      }
+    }
+  },
+  "variables": {
+    "deploymentUrlBase": "https://raw.githubusercontent.com/DFE-Digital/bat-platform-building-blocks/master/templates/"
+  },
+  "resources": [
+    {
+      "name": "container-instances",
+      "type": "Microsoft.Resources/deployments",
+      "apiVersion": "2017-05-10",
+      "properties": {
+        "mode": "Incremental",
+        "templateLink": {
+          "uri": "[concat(variables('deploymentUrlBase'), 'container-instances.json')]",
+          "contentVersion": "1.0.0.0"
+        },
+        "parameters": {
+          "containerInstanceName": {
+            "value": "[parameters('containerInstanceName')]"
+          },
+          "containerName": {
+            "value": "[parameters('containerName')]"
+          },
+          "imageName": {
+            "value": "[parameters('imageName')]"
+          },
+          "environmentVariables": {
+            "value": [
+              {
+                "name": "RAILS_ENV",
+                "value": "[parameters('railsEnv')]"
+              },
+              {
+                "name": "AUTHENTICATION_TOKEN",
```

Comment:
I just had a look through the code for how this is used, and it's only used in authenticating requests for API V1. If we remove this then I *think* the authentication token will be blank, and while we don't expose a way to reach API V1 here, if we ever did using this configuration we'd be leaking data.

At the risk of being paranoid, I say we just leave it here. What we really outta be doing is setting this kind of thing to a secure string (read: random) by default, with overrides per environment.

Reply:
We are already using this configuration in rest of the other ARM templates. Authentication token value goes in application settings which takes only string values.